tests/integration/cli/test_rituals.py

```
@@ -538,3 +544,48 @@ def test_showing_a_ritual_that_cannot_be_loaded_is_a_usage_error(
 
         assert exit_code == _USAGE_EXIT
         assert "a_module_that_does_not_exist" in capsys.readouterr().err
+
+    # "no rituals found (create a rituals.py or a rituals/ package)" printed
+    # next to a directory called `rituals` sends the reader to write the file
+    # they have already written. The missing piece is one empty `__init__.py`.
+    @staticmethod
+    def test_a_rituals_directory_that_is_not_a_package_says_what_is_missing(
```

Ruff (0.16.1)

[warning] Missing return type annotation for staticmethod `test_a_rituals_directory_that_is_not_a_package_says_what_is_missing`

Add return type annotation: `None`

(ANN205)